pkg/services/annotations/accesscontrol/accesscontrol.go

```diff
@@ -131,13 +134,17 @@ func (authz *AuthService) dashboardsWithVisibleAnnotations(ctx context.Context,
 		})
 	}
 
+	section := authz.cfg.Raw.Section("annotations")
+	key := "dashboards_with_visible_annotations_search_dashboards_page_limit"
+	limit := section.Key(key).MustInt64(1000)
```

Comment:
could we initialize this instead in the `NewAuthService` function, so we don't have to re-read it on every call?

Reply:
yes!